src/styles/utilities/cursor/types.ts

```diff
@@ -0,0 +1,5 @@
+import {CSSProperties} from 'react';
+
+type CursorStyles = Record<string, Partial<Pick<CSSProperties, 'cursor'>>>;
```

Comment:
@BartoszGrajdek Could we replace `string` with all the cursor options (`cursorDefault`, `cursorDisabled`, etc) to make the type more strict?

Reply:
@fabioh8010 Of course, if we want to be that strict let's make it so :)